plugins/jobs/plugin_tests/jobsSpec.js

```diff
@@ -454,5 +457,44 @@ $(function () {
               expect(fired).toBe(true);
           });
       });
+
+        it('Test show all jobs option', function () {
```

Comment:
`it('Job list widget show all jobs.', ...` would be a more consistent test name.